code/modules/research/designs/weapon_designs.dm

```diff
@@ -59,6 +59,18 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
 
+/datum/design/fortynr
+	name = "40nr magazine (Lethal)"
+	desc = "Magazine designed for SP-8 pistol. Bullets have good stopping power."
+	id = "fortynr"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = SHEET_MATERIAL_AMOUNT * 10)
+	build_path = /obj/item/ammo_box/magazine/sp8
```

Comment:
All other comments aside; this is setting a dangerous precedent. We currently control ballistic access and usability in two ways. The gun itself and its exclusivity. And magazines. 

Getting another .38 is difficult, and largely a unique toy for the detective. Once we start introducing more weapons like the .38, we start diminishing the unique quality of that weapon.

In order to partially justify the weapons existence you've severely gimped its actual practicality by making it very weak against meaningful human threats (dudes in armor). Because by not doing that, the gun is just a disabler that kills you and you can tactically reload it from your well of ammo.

We don't have the design space for encouraging more ballistic additions that work like this than we what we have. And it is evident by having to make it the way you have. I just don't think this is a good idea.